src/wp-includes/link-template.php

```diff
@@ -2785,8 +2785,8 @@ function get_the_posts_navigation( $args = array() ) {
 			)
 		);
 
-		$next_link = get_previous_posts_link( $args['next_text'] );
-		$prev_link = get_next_posts_link( $args['prev_text'] );
+		$next_link = get_next_posts_link( $args['next_text'] );
+		$prev_link = get_previous_posts_link( $args['prev_text'] );
```

Comment:
As mentioned in the documentation:
https://developer.wordpress.org/reference/functions/get_next_posts_link/
Because post queries are usually sorted in reverse chronological order, `get_next_posts_link()` usually points to older entries (toward the end of the set) and
`get_previous_posts_link() `usually points to newer entries (toward the beginning of the set).